tests/unit/Modeler.spec.js

```diff
@@ -0,0 +1,8 @@
+import Modeler from '../../src/components/Modeler.vue';
+
+describe('registerInspectorExtension', () => {
+    it('adds config with a container', () => {
+        // Placeholder
+        expect(typeof Modeler.created).toBe('function')
+    })
+})
```

Comment:
You can remove this file for now as it's causing our CI to fail, but you've definitely highlighted our need for unit tests and we should make this a priority going forward!